.github/workflows/main.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
       - name: Python linter
         shell: bash -l {0}
-        run: flake8 sequence_processing_pipeline/ setup.py sequence_processing_pipeline/scripts/sequence-processing-pipeline
+        run: flake8 sequence_processing_pipeline/ setup.py
```

Comment:
I thought it best to remove the command-line app since it's just an empty shell. Once everything has matured, we could consider bring it back easily enough.